app/controllers/admin/dashboard_controller.rb

```diff
@@ -9,9 +9,9 @@ class Admin::DashboardController < Admin::BaseController
   def index
     #@topics = Topic.mine(current_user.id).pending.page params[:page]
 
-    if current_user.is_admin? || current_user.is_agent?
+    if (current_user.is_admin? or current_user.is_agent?) and (forums? or tickets?)
```

Comment:
I suppose it really is up to you, but it would be better to keep it consistent across the whole project.

#375 adds the cop that checks for this style violation, and also changes other occurrences of `and` or `or`.